desktop/appcontainer/AppContainer.js

```diff
@@ -129,6 +130,17 @@ const appContainerView = hoistCmp.factory({
     }
 });
 
+const bannerList = hoistCmp.factory({
+    render({model}) {
+        const {bannerModels} = model.bannerSourceModel;
+        return (isEmpty(bannerModels)) ?
+            null :
+            fragment({
+                items: bannerModels.map(model => banner({model, key: model.xhId}))
+            });
```

Comment:
I didn't think it was important by any means to avoid returning an empty fragment, but it seemed slightly cleaner (and hopefully not worse in any way).

Reply:
Makes sense! Unless there are any objections, I might change this to an early return rather than a ternary - find those easier to read